integration_tests/e2e/index.cy.ts

```diff
@@ -6,18 +6,16 @@ context('Index page', () => {
     cy.resetBasicStubs()
   })
 
-  it('should show expected header and footer elements', () => {
+  it('User name visible in fallback header', () => {
     cy.signIn()
-
-    const homePage = Page.verifyOnPage(IndexPage)
-
-    homePage.headerUserName.should('contain.text', 'J. Smith')
-    homePage.activeCaseload.should('contain.text', 'Moorland (HMP & YOI)')
-
-    homePage.footerLinks.spread((...links) => {
-      expect(links).to.have.lengthOf(2)
-      expect(links[0]).to.contain('Get help')
-      expect(links[1]).to.contain('Terms')
-    })
+    cy.task('stubComponentsFail')
```

Comment:
could you add `cy.task('stubComponentsFail')` to the the end of `Cypress.Commands.add('resetBasicStubs', () => {…}` in `integration_tests/support/commands.ts`? so that the other pages we're testing use the fallback header & footer